examples/source/interactivity-dynamic-content/Subscription_Settings/index.html

```diff
@@ -112,17 +112,20 @@
           {
             "value": "watching",
             "isSelected": false,
-            "text": "Watching"
+            "text": "Watching",
+            "imgUrl": "/images/watching.jpg"
```

Comment:
Do this images exist?

Reply:
The images that are actually used in the code to exist, the ones in the comments just shortened fake ones for the sake of brevity/readability. If you think the value having them be the actual your URLs is higher I can change it

Reply:
got it